### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by izeye**
This PR fixes some typos and polishes trivial stuff.

### CODE REVIEW

File: spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/autoconfigure/LocalDevToolsAutoConfiguration.java [hunk 1]
```diff
@@ -191,14 +191,11 @@ public boolean supportsSourceType(@Nullable Class<?> sourceType) {
 
 		@Override
 		public void onApplicationEvent(ApplicationEvent event) {
-			if (event instanceof ContextRefreshedEvent) {
+			if (event instanceof ContextRefreshedEvent
```
Comment (snicoll) [inline spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/autoconfigure/LocalDevToolsAutoConfiguration.java]: Can we please keep the two ifs please? (and combine the two conditions for the second if). It reads better IMO

Comment (izeye) [reply spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/autoconfigure/LocalDevToolsAutoConfiguration.java]: @snicoll I thought that the two `if` statements doing the same thing are unusual as it looks to me as follows:

```
if (condition1) {
  doSomething();
}
if (condition2) {
  doSomething();
}
```

WDYT? If you still prefer as-is, please let me know. Then I'll revert it.

Comment (snicoll) [reply spring-boot-project/spring-boot-devtools/src/main/java/org/springframework/boot/devtools/autoconfigure/LocalDevToolsAutoConfiguration.java]: Thanks for the feedback. There are but the proposal condition is a big fat IMO. I am not really leaning one way or the other actually, let me checkout the code in an IDE.

**Comment by snicoll** [comment]
Thanks again @izeye!

**Pull request closed without merging**